modules/Bio/EnsEMBL/Compara/GeneTreeNode.pm

```diff
@@ -306,17 +307,37 @@ sub root {
 =cut
 
 sub is_leaf {
-  my $self = shift;
-
+    my $self = shift;
     my $child_count = $self->get_child_count;
-    if ( $child_count == 0 ) {
-        return 1;
-    } elsif ( $child_count == 1 && $self->tree->tree_type eq 'supertree' ) {
+    return 1 if ($child_count == 0);
```

Comment:
Is it silly to find this code much more readable than `return 1 unless $child_count`? :sweat_smile: